src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -306,7 +312,7 @@ public function dump(array $options = [])
 
 use Symfony\Component\DependencyInjection\Dumper\Preloader;
 
-require dirname(__DIR__, 3).'/vendor/autoload.php';
+require $autoloadFile;
```

Comment:
now with proper detection of `autoload.php` location

Reply:
Do we have any test covering the dumping of this file ?

Reply:
yes; not precisely this line but the `$classes[] = ...` one below